interface/wx/platinfo.h

```diff
@@ -270,6 +270,9 @@ class wxPlatformInfo
     /**
         Returns the global wxPlatformInfo object, initialized with the values
         for the currently running platform.
+
+        Note that this function is thread-safe, i.e. it can be called
+        concurrently from multiple threads without locking.
```

Comment:
The implementation is dependent on `wxUSE_THREADS`, but I would guess that a user program could be threaded even on a wx library that is not configured to use threads, so maybe it is better to mention that this is only handled by wx if `wxUSE_THREADS` is set, and otherwise is the responsilibity of the caller. (I imagine most people are not changing this though).